gdal/frmts/raw/envidataset.cpp

```diff
@@ -1430,12 +1431,26 @@ bool ENVIDataset::ProcessMapinfo( const char *pszMapinfo )
 
 {
     char **papszFields = SplitList(pszMapinfo);
+    char *units = NULL;
+    double rotation = 0.0;
     const int nCount = CSLCount(papszFields);
 
     if( nCount < 7 )
     {
         CSLDestroy(papszFields);
         return false;
+    } else if( nCount > 10 )
+    {
+        // Retrieve named values
+        for (int i=10; i<nCount; ++i)
+        {
+            if (strncmp(papszFields[i], "units=", strlen("units=")) == 0)
+            {
+                units = papszFields[i] + strlen("units=");
+            } else if (strncmp(papszFields[i], "rotation=", strlen("rotation=")) == 0) {
+                rotation = CPLAtof(papszFields[i] + strlen("rotation=")) * (M_PI/180) * -1;
```

Comment:
I didn't understand why it was necessary to multiply by `-1`, but the rotation went the wrong way if I didn't.